pkg/replica/diff_disk.go

```diff
@@ -304,10 +308,15 @@ func (d *diffDisk) UnmapAt(unmappableDisks []string, length uint32, offset int64
 	if endSectorOffset != 0 {
 		length -= uint32(endSectorOffset)
 	}
-	if length <= 0 {
+	if length == 0 {
```

Comment:
Is it possible that length is smaller than 0?

Reply:
Why cannot this help handle the negative `length` case?

Reply:
Because the length is uint32, it overflows when the result is negative.
The original check was fine, but it could not capture the negative case, so I changed it to `==` instead of `<=`

Reply:
OK. Using `<=` rather than `==` is due to that the data type of `length` is unsigned.